wire-ios-data-model/Source/ManagedObjectContext/CoreDataStack+Backup.swift

```diff
@@ -83,17 +83,16 @@ extension CoreDataStack {
     ///   - accountIdentifier: identifier of account being backed up
     ///   - applicationContainer: shared application container
     ///   - dispatchGroup: group for testing
-    ///   - encryptionKeys: EAR encryption keys
+    ///   - databaseKey: EAR database key
     ///   - completion: called on main thread when done. Result will contain the folder where all data was written to.
     public static func backupLocalStorage(
         accountIdentifier: UUID,
         clientIdentifier: String,
         applicationContainer: URL,
         dispatchGroup: ZMSDispatchGroup? = nil,
-        encryptionKeys: EncryptionKeys? = nil,
+        databaseKey: VolatileData? = nil,
```

Comment:
was this change not already in ? I thought EncryptionKeys were removed completly @johnxnguyen

Reply:
Not sure what happened here, but I merged the base branch back in and it's no longer showing in the diff.